go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/rode/rode
 
 go 1.16
 
+replace github.com/rode/grafeas-elasticsearch => /Users/parker/Developer/grafeas-elasticsearch
```

Comment:
I'll remove this when I'm done testing the grafeas-elasticsearch changes locally.  tests will fail until this is removed.